src/pages/ProfilePage.tsx

```diff
@@ -120,9 +159,42 @@ const ProfilePage: FC = () => {
                 subtitle="+"
                 onClick={() => history.replace('/home/scan-qr')}
               />
-            </DetailCardsStyled>
+            </DetailCardsStyled>            
+            <HorizontalLine style={{marginTop:"150px"}}/>
+            <IonRow>
+              <IonCol size="10" offset="1" className="ion-text-center ion-padding">              
+                <Title fontSize={FontSize.XS} style={{fontFamily:"Open Sans",color:"#707070"}}>Available NASCAR balance:</Title> 
+              </IonCol>
+            </IonRow>      
+            <IonRow style={{marginTop:"10px"}}>
+              <IonCol size="10" offset="1" className="ion-text-center">              
+                <Title fontSize={FontSize.XXL} fontWeight={FontWeight.BOLD} style={{fontFamily:"Open Sans",color:"#8AC827"}}>${(Number(wallet?.balance)/10**8).toFixed(2)}</Title> 
+              </IonCol>
+            </IonRow>          
+            <HorizontalLine/>
+            </>
           )}
         </IonGrid>
+        {process.env.REACT_APP_PAPER_COINS_FAUCET_SENDER_PASSPHRASE && process.env.REACT_APP_PAPER_COINS_FAUCET_SENDER_PASSPHRASE.trim().length > 0 
+          && Number(wallet?.balance)/10**8 < Number(process.env.REACT_APP_PAPER_COINS_MIN_AVAILABLE_TO_ACTIVATE_FAUCET!) && (
```

Comment:
You can simplify this with optional chaining: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

```suggestion
        {process.env.REACT_APP_PAPER_COINS_FAUCET_SENDER_PASSPHRASE?.trim().length > 0 
          && Number(wallet?.balance)/10**8 < Number(process.env.REACT_APP_PAPER_COINS_MIN_AVAILABLE_TO_ACTIVATE_FAUCET!) && (
```

Reply:
I think that would trigger an error, becuase would get "undefined" value, so undefined.trim() will trigger an exception

Reply:
Optional chaining `?.` would save you with returning `undefined` before even reaching `.trim()` function.